sdk/src/azure/core/az_mqtt5_rpc.c

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/core/az_mqtt5_rpc.h>
+#include <azure/core/az_result.h>
+#include <azure/core/az_span.h>
+#include <azure/core/internal/az_log_internal.h>
+#include <azure/core/internal/az_precondition_internal.h>
+#include <stdio.h>
+
+#if defined(TRANSPORT_MOSQUITTO)
+#include <mosquitto.h>
+#endif
+
+#include <azure/core/_az_cfg.h>
+
+static const az_span model_id_key = AZ_SPAN_LITERAL_FROM_STR("{serviceId}");
+static const az_span command_name_key = AZ_SPAN_LITERAL_FROM_STR("{name}");
+static const az_span executor_client_id_key = AZ_SPAN_LITERAL_FROM_STR("{executorId}");
+static const az_span invoker_client_id_key = AZ_SPAN_LITERAL_FROM_STR("{invokerId}");
+
+AZ_NODISCARD int32_t
+_az_rpc_calculate_substitution_length(az_span format, az_span key, az_span value);
```

Comment:
Should at least some of these `_az' functions be made `static` (or even `AZ_INLINE` for those used in a single place)?